apps/wakunode2/external_config.nim

```diff
@@ -304,10 +305,26 @@ type
         name: "filternode" }: string
 
       filterTimeout* {.
-        desc: "Timeout for filter node in seconds.",
+        desc: "Filter clients will be wiped out if not able to receive push messages within this timeout. In seconds.",
```

Comment:
I'm a bit confused.

Does this means that a Filter node will try to push messages to a client for 4 hours before giving up?

If a client has to ping every 5m or the sub is dropped, how does that make sense? I don't understand.

Reply:
Well, its truely confusing, hence the thread: https://discord.com/channels/1110799176264056863/1192407282420551742

So filterTimeout is for legacy filter only. 
There is the point where we started diverge the two kind of filter so much in terms of behavior.
Only solution I see is to soon remove legacy filter functionality.

Reply:
I can't wait to remove legacy filter it's gonna be so much cleaner!